src/main/java/de/doubleslash/keeptime/model/LicenceTableRow.java

```diff
@@ -0,0 +1,41 @@
+// Copyright 2019 doubleSlash Net Business GmbH
+
+package de.doubleslash.keeptime.model;
```

Comment:
das sollte in das view package verschoben werden (in unter package), da es nur dort benötigt wird / zur view benötigt wird und nichts mit unserem eigentlich Model zu tun hat.

Reply:
Ich dachte im model sind daten gespeichert, die der view dann anzeigt? genau das tut doch LicenceTableRow ...